src/functions-and-arrays.js

```diff
@@ -1,19 +1,64 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if(num1 > num2){
+  return num1;
 
+  }else{
```

Comment:
You forgot one step here, "should return greater of two arguments - if the second argument greater".
I see that you returned num2 later but the idea here was to check,
```
if(num1 < num2){
   return num2;
```
and then, return num1 or num2, like:
```
else {
return num1, num2;
}
```